var/spack/repos/builtin/packages/iegenlib/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# -----------------------------------------------------------------------------
+# Author: Derick Huth <derick.huth@utah.edu>
+# -----------------------------------------------------------------------------
+#   Note: This package was created by the ctop reaserch group at
+#         the University of Utah, Shool of Computing and is unifiliated
+#         with IEGenLib
+# -----------------------------------------------------------------------------
+from spack import *
+
+
+class Iegenlib(AutotoolsPackage):
+    """Inspector/Executor Generation Library for manipulating sets
+       and relations with uninterpreted function symbols. """
+
+    homepage = "http://github.com/CompOpt4Apps"
+    git      = "https://github.com/CompOpt4Apps/IEGenLib.git"
+
+    version('master', branch='master')
+
```

Comment:
Why doesn't `autoreconf` need to be run, as is typical when checking Autotools packages out of a git repo?  Note that checking the `./configure` script into the Git repo is a bad idea (search around for why).

Reply:
IEGenLib is a cmake package with it's own ./confgiure script. It otherwise works like an autotools package.